mantle/kola/tests/misc/network.go

```diff
@@ -273,6 +283,17 @@ var (
           primary_ip=%s
           secondary_ip=%s
 
+          connections_folder=$(NetworkManager --print-config | sed -nr "/^\[keyfile\]/ { :l /^path[ ]*=/ { s/.*=[ ]*//; p; q;}; n; b l;}")
+          if [[ -z $connections_folder ]]; then
+            connections_folder=/etc/NetworkManager/system-connections
+          fi
+          rm -rf ${connections_folder}/*
```

Comment:
@RamLavi  Not sure this is the right location, I mean
If the folder exits with configuration it will delete all of the content.
The delete should be right before the copy new content

Reply:
I put it here to prevent a race between the script creating the veths and the initialization of the connections via nmcli..
If I don't remove it, then the connection will recreate from the folder, and the veth dhcp will fail (the bond and primary veth use the same ip).
Since the veth creation is just a pre script that lays the foreground  used by the setup-ovs script, we need to make sure the old connection don't get in our way..
I hope I explained well.

Reply:
But at the start of the script we are delete all the folder.. 
Line 5: 
rm -rf /etc/NetworkManager/system-connections/*

So we are doing this twice?

Reply:
both scripts `create-veth-pairs` and `setup-ovs` is removing the directory before starting..
`create-veth-pairs` runs first and we need to run it to avoid the dhcp race over the ips.
`setup-ovs` runs second, so basically we don't need to "re-delete" it, but since we want to check `setup-ovs` as a whole, then I kept it.